src/lib/pages/backoffice/Home/lists/OverbookedDocumentsList/state/actions.js

```diff
@@ -13,7 +13,7 @@ export const fetchOverbookedDocuments = () => {
 
     try {
       const response = await documentApi.list(
-        documentApi.query().overbooked().qs()
+        documentApi.query().overbooked().sortBy('-loan_requests').qs()
```

Comment:
Question: What does the minus sign (`-`) mean? Is it for the order?

Reply:
Yes, the trick here is that if you use React-SearchKit, then it is hiding this complexity. You will have the sort order `asc` or `desc`.
If instead you directly use Invenio APIs, which is using Elasticsearch behind, then you have have `fieldname` for ASC and `-fieldname` for DESC (unless configured in a different way).